emanifest-py/README.md

```diff
@@ -17,7 +17,7 @@
   - [Methods](#methods)
   - [Help](#help)
 - [Contact](#contact)
-- [License](LICENSE.txt)
+- [License](LICENSE)
```

Comment:
I tested this without the .txt extension and didn't get it to link correctly to the license page. I think we should keep the extension - please let me know if I messed up and it's fine

Reply:
TLDR we can change it back if you want, just let me know. 

hmmm that's weird. I can't imagine the file extension is the problem. 

The only reason I changed it  was b/c I was debugging an error about "LICENSE.txt" being an unrecognized format or something but it turns out that file wasn't the problem at all.

Reply:
Yeah, let's go ahead and change it back. If it catches fire in the next release we know where to start with the fix.

Reply:
@wiljnichepa 
If it sets you mind at ease, you can test that the README links to the LICENSE file on this branch here
https://github.com/dpgraham4401/e-manifest/tree/publish_workflow/emanifest-py

Doesn't appear to have any problems. 

If I don't hear back from you, i'll change it to LICENSE.txt